.github/workflows/build_and_test.yml

```diff
@@ -12,6 +12,7 @@ env:
   RISC0_DEV_MODE: 1
   ENABLE_HEADER_CHAIN_PROVER: 1
   RUST_MIN_STACK: 33554432
+  IS_CI: 1
```

Comment:
https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables#default-environment-variables

Can we use `CI`, it's always set to `true`